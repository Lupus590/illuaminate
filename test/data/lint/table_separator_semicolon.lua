-- config: (lint (only format:table-separator) (table-separator semicolon))

local _ = { a, b; c, d; }
