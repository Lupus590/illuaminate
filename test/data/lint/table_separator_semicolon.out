table_separator_semicolon.lua:[3:14-3:14]: Table fields should be separated by a semicolon. [format:table-separator]
   │
 3 │ local _ = { a, b; c, d; }
   │              ^
table_separator_semicolon.lua:[3:20-3:20]: Table fields should be separated by a semicolon. [format:table-separator]
   │
 3 │ local _ = { a, b; c, d; }
   │                    ^
No errors and 2 warnings
 -- config: (lint (only format:table-separator) (table-separator semicolon))
 
-local _ = { a, b; c, d; }
+local _ = { a; b; c; d; }
 
